conf/getConf.js

```diff
@@ -1,17 +1,19 @@
-const { config: browserStackRemoteConf } = require('./browserstack.remote.conf')
-const { config: browserStackLocalConf } = require('./browserstack.local.conf')
-const { config: localConf } = require('./local.conf')
+const { config: defaultBrowserStackConf } = require('./browserstack.conf')
+const { config: defaultBrowserStackLocalConf } = require('./browserstack.local.conf')
+const { config: defaultLocalConf } = require('./local.conf')
 
-exports.getConf = () => {
+exports.getConf = ({
```

Comment:
i don't see when we are going to use this yet, but maybe in the future 👍

Reply:
I think I have something in my mind like `extends` instead, but let's keep it as an option.

It will be useful to pass local configs before publishing them (and not having to modify them inside node_modules
)